build-logic/documentation/src/main/groovy/gradlebuild/docs/FindBrokenInternalLinks.java

```diff
@@ -49,29 +51,66 @@
 @CacheableTask
 public abstract class FindBrokenInternalLinks extends DefaultTask {
 
+    // <<groovy_plugin.adoc#groovy_plugin,Groovy>>
     private final Pattern linkPattern = Pattern.compile("<<([^,>]+)[^>]*>>");
+    // groovy_plugin.adoc#groovy_plugin,Groovy
     private final Pattern linkWithHashPattern = Pattern.compile("([a-zA-Z_0-9-.]*)#(.*)");
```

Comment:
I think by default, asciidoc doesn't enforce this because this isn't the "proper" way to cross-reference documents. Ideally we should be using [`xref:`](https://docs.asciidoctor.org/asciidoc/latest/macros/inter-document-xref/)

I will add this as a TO DO on my cleanup list.